android/app/src/main/java/com/zulipmobile/notifications/GCMPushNotifications.java

```diff
@@ -36,6 +37,7 @@
 
     private static final String CHANNEL_ID = "default";
     private static final int NOTIFICATION_ID = 435;
+    private static final String NOTIFICATION_SOUND_PATH = "/raw/zulip";
```

Comment:
Hmm. I'd prefer the `R.raw.zulip` form, because it's more transparent to tools, like Android Studio.

That form [is documented](https://developer.android.com/reference/android/content/ContentResolver#openAssetFileDescriptor(android.net.Uri,%20java.lang.String,%20android.os.CancellationSignal)), with no sign of deprecation. Seems like it should work.

You mention in the commit message an error about `android.resource://com.zulipmobile.debug2131165184`. That is missing a slash. Perhaps that's an unrelated issue?